predeploy.py

```diff
@@ -6,16 +6,20 @@
 
 
 files = ["requirements.txt", "Procfile"]
+rfiles = ["Aptfile", "init.R"]
 
-app_path = os.path.join(
-    "apps",
-    APPNAME_TO_DIRECTORY.get(os.environ["DASH_APP_NAME"], os.environ["DASH_APP_NAME"]),
-)
+app_name = os.environ["DASH_APP_NAME"]
+
+app_path = os.path.join("apps", APPNAME_TO_DIRECTORY.get(app_name, app_name))
 
 for f in files:
     shutil.copyfile(os.path.join(app_path, f), f)
 
 if "DOKKU_SCALE" in os.listdir(app_path):
     shutil.copyfile(os.path.join(app_path, "DOKKU_SCALE"), "DOKKU_SCALE")
 
+if "-" in app_name and app_name.split("-")[0] == "dashr":
+    for rfile in rfiles:
+        shutil.copyfile(os.path.join(app_path, rfile), rfile)
+
 subprocess.run("python -m pip install -r requirements.txt".split(" "))
```

Comment:
@rpkyle for `R`, this last line will fail, so might wanna put it inside a condition so that it is only executed for python apps:

```
if "-" in app_name and app_name.split("-")[0] == "dashr":
    for rfile in rfiles:
        shutil.copyfile(os.path.join(app_path, rfile), rfile)
else:
    subprocess.run("python -m pip install -r requirements.txt".split(" "))
```

This is not really a very clean way IMHO, but it should do until we refactor `predeploy.py` to have different code-paths for `R` and `python` apps.